demo/helper-scripts/demo-pod-with-discovery.json.parsec.template

```diff
@@ -0,0 +1,25 @@
+{
+    "apiVersion": "v1",
+    "kind": "Pod",
+    "metadata": {
+        "name": "demo-parsec-NUM"
+    },
+    "spec": {
+        "containers": [
+            {
+                "image": "intelsdi/node-feature-discovery-APP",
```

Comment:
Could we host these container images together with the main NFD image on quay.io? e.g. in the incubator org